src/live-updates.ts

```diff
@@ -16,9 +17,11 @@ export class LiveUpdates {
 
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   private mergeGraphQL(initial: Argument, locale: string, updated: Entity): Argument {
-    // TODO: https://contentful.atlassian.net/browse/TOL-1000
-    // TODO: https://contentful.atlassian.net/browse/TOL-1022
-    return initial;
+    if ((initial as any).__typename === 'Asset') {
```

Comment:
`Argument` can also be an array currently, maybe we should adjust this check here

Reply:
Btw I think they handle the type in the starter template like this

```
{
    sys: { id: string };
    __typename: string;
    [k: string]: any;
}
```